docs/content/docs/test/Runner.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Test Runner
+type: docs
+---
+
```

Comment:
Add specific internal tag. @eugene-krivobokov help me, please.

Reply:
`{{< avito >}}`

https://avito-tech.github.io/avito-android/docs/contributing/docs/#internal-links

Reply:
thanks, never used it